bin/kano-world-hook

```diff
@@ -26,48 +34,73 @@ if __name__ == '__main__' and __package__ is None:
 
 from kano.logging import logger
 
+
+# TODO: Make these return codes publicly available
+SUCCESS = 0
+E_INVALID_URL = -1
+E_INVALID_SCHEME = -2
+E_SCHEME_NOT_FOUND = -3
+E_SCHEME_EXECUTION_FAILED = -4
+
+
 # TODO: Put kano-dialogs where appropriate
 
-if len(sys.argv) < 2:
-    msg = "{} run with no arguments".format(sys.argv[0])
-    logger.error(msg)
-    sys.exit(0)
+def main(scheme_url, **args):
+    # Parse the URL
+    try:
+        url_data = scheme_url.split(":")
```

Comment:
Nice find Albert. I didn't want to make too drastic a change for this fix but we should definitely migrate to use that after release.